Aliucord/src/main/java/com/aliucord/coreplugins/plugindownloader/PluginFile.kt

```diff
@@ -37,11 +37,13 @@ internal class PluginFile(val plugin: String) : File("${Constants.PLUGINS_PATH}/
                         PluginManager.stopPlugin(plugin)
                         PluginManager.unloadPlugin(plugin)
                     }
-                    PluginManager.loadPlugin(Utils.appContext, this)
-                    if (PluginManager.isPluginEnabled(plugin))
-                        PluginManager.startPlugin(plugin)
-                    else
-                        PluginManager.enablePlugin(plugin)
+                    if (!PluginManager.isSafeModeEnabled()) {
+                        PluginManager.loadPlugin(Utils.appContext, this)
+                        if (PluginManager.isPluginEnabled(plugin))
+                            PluginManager.startPlugin(plugin)
+                        else
+                            PluginManager.enablePlugin(plugin)
+                    }
```

Comment:
You should display a toast here if safe mode is on, "plugin downloaded but not loaded due to safe mode" or something